src/docker/registry.nim

```diff
@@ -462,16 +471,46 @@ proc manifestGet*(image:  DockerImage,
     )
   return newDockerDigestedJson(response.body(), image.imageRef, accept, kind)
 
-proc layerGet*(image:  DockerImage,
-               accept: string,
-               use =   RegistryUse.ReadOnly,
-               ): DockerDigestedJson =
+proc layerGetString*(image:  DockerImage,
+                     accept: string,
+                     use =   RegistryUse.ReadOnly,
+                     ): string =
   let
-    kind          = CONTENT_TYPE_MAPPING[accept]
     (_, response) = image.request(
       use        = use,
       httpMethod = HttpGet,
       path       = "/blobs/" & image.imageRef,
       accept     = accept,
     )
-  return newDockerDigestedJson(response.body(), image.imageRef, accept, kind)
+  return response.body()
+
+proc layerGetJson*(image:  DockerImage,
+                   accept: string,
+                   use =   RegistryUse.ReadOnly,
+                   ): DigestedJson =
```

Comment:
```suggestion
proc layerGetJson*(
    image:  DockerImage,
    accept: string,
    use =   RegistryUse.ReadOnly,
): DigestedJson =
```

Reply:
inline with our style discussion yesterday

Reply:
most of chalk uses hanging indents for function signatures now. when we switch to auto-formatter we can switch to something else then. for now we can keep as-is?